### DISCUSSION TITLE
Unable to use on WP site running on PHP 7.4

### DISCUSSION THREAD

**Discussion by lopadz** [spatie/ray#763] [General]
~~While the WP Ray plugin lists being compatible with PHP 7.4, when trying to activate it, the following error appears:~~

```
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.0".
```

Wrong repo. In case someone was wondering, this issue has been reported here: https://github.com/spatie/wordpress-ray/issues/55

**Comment by freekmurze** [comment]
For support for older PHP versions, you can use v1.5.2 of the plugin.
